platform/src/test/kotlin/org/stellar/anchor/platform/config/PropertyClientsConfigTest.kt

```diff
@@ -18,12 +18,31 @@ class PropertyClientsConfigTest {
     configs = PropertyClientsConfig()
     errors = BindException(configs, "config")
   }
+
   @Test
-  fun `test valid custodial client`() {
+  fun `test postConstruct`() {
+    val config = ClientConfig()
+    config.name = "sampleName"
+    config.signingKey = "sampleSigningKey"
+    config.domain = "sampleDomain"
+
+    configs.clients.add(config)
+    configs.postConstruct()
+
+    Assertions.assertEquals(config.signingKey, config.signingKeys.first())
```

Comment:
nit: Can we check that there are no other values in `signingKeys` besides the `signingKey`?